src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/Ceil.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\Doctrine\ORM\Query\AST\Functions;
+
+/**
+ * Implementation of PostgreSQL CEIL().
+ *
+ * @see https://www.postgresql.org/docs/17/functions-math.html
+ * @since 3.1
```

Comment:
Please change this to 3.2 as this is the next feature version where it will be included.

Reply:
Done